src/http.rs

```diff
@@ -33,24 +31,46 @@ pub fn get_client() -> Client<Connector, hyper::Body> {
 pub fn fetch_sync_string(module_name: &str) -> DenoResult<String> {
```

Comment:
It would be very nice to have a unit test here... Not necessary but can you try uncommenting the test below? Maybe it has spontaneously been fixed...

Reply:
Seems that test would still get stuck

Reply:
I think there is a bug similar to #919 here, adding a `println!` in the test unblocks the test

Reply:
Ugh. Ok. Good find - I will look into it tomorrow

Reply:
@ry Actually, tests start to run again by removing the outer `tokio_util::block_on`. There is already another `block_on` inside of `fetch_sync_string`